nmigen/lib/roundrobin.py

```diff
@@ -0,0 +1,50 @@
+from .. import *
+
+
+__all__ = ["RoundRobin"]
+
+
+class RoundRobin(Elaboratable):
+    """Round-robin scheduler.
+
+    For a given set of requests, the round-robin scheduler will
+    grant one request.
+
+    Parameters
+    ----------
+    width : int
+        Number of requests.
+
+    Attributes
+    ----------
+    requests : Signal(width), in
+        Set of requests.
+    grant : Signal(range(width)), out
+        Number of the granted request.
+    en : Signal, in
```

Comment:
Why not let the end user of the module apply `EnableInserter` instead?

Reply:
Good call.